.github/matrix-commitly.yml

```diff
@@ -4,15 +4,27 @@ build-packages:
   os: ubuntu-22.04
   package: deb
   check-manifest-file: ubuntu-22.04-amd64.txt
+# Alpine
+- label: alpine
+  os: ubuntu-22.04
```

Comment:
should be `alpine`?

Reply:
@chronolaw we do cross compile (host is ubuntu and target is alpine) for alpine build, so this is correct

Reply:
OK, thanks for your explain.